services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/flag_filters.tsx

```diff
@@ -58,13 +59,14 @@ const FlagFilters = ({ activities, filterActivities, flagFilters, handleFlagFilt
 
   const filteredActivities = filterActivities(FLAG_FILTERS)
 
-  const flagRows = ['alpha', 'beta', 'gamma', 'production', 'archived', 'private'].map(ac =>
+  // TODO: flag display
```

Comment:
Adding these comments so that tracking down display name code locations is easier.